javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,15 @@
+const fullName = 'Cody Miller';
+const isCool = true;
+const totalPets = 1;
+let undefinedVar;
+const undVar2 = undefined;
+console.log(`value of fullName: ${fullName}`);
+console.log(`value of isCool: ${isCool}`);
+console.log(`value of totalPets: ${totalPets}`);
+console.log(`typeof fullName: ${typeof fullName}`);
+console.log(`value of isCool: ${typeof isCool}`);
+console.log(`typeof totalPets: ${typeof totalPets}`);
+console.log(`value of unknown: ${null}`);
+console.log(`typeof unknown: ${typeof null}`);
+console.log(`value of never: ${typeof undefinedVar}`);
+console.log(`typeof never: ${typeof undVar2}`);
```

Comment:
You can do `console.log`s like this, but once we get into objects, you're going to get some weird things if you try to log them like this. So it's fine for the most part, but it won't work with objects. So I would probably suggest doing them the other way, by just passing in 2 arguments.